qiskit/util.py

```diff
@@ -95,3 +95,19 @@ def _has_connection(hostname, port):
         return True
     except Exception:  # pylint: disable=broad-except
         return False
+
+
+def _to_tuple(values):
+    """
+    Return the input, sorted, and as a tuple.
```

Comment:
How come "sorted"?

Reply:
Thank you for pointing that out. It was a typo.